src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,53 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    public static final DateTimeFormatter DATA_FORMATTER
+            = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
+    private static final int DATE_INDEX = 0;
+    private static final String SEPARATOR = " ";
+    private static final int NAME_INDEX = 1;
+    private static final int HOUR_INDEX = 2;
+    private static final int WAGE_INDEX = 3;
+    private static final String DASH = " - ";
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        LocalDate startDate = LocalDate.parse(dateFrom, DATA_FORMATTER);
+        LocalDate endDate = LocalDate.parse(dateTo,DATA_FORMATTER);
```

Comment:
```suggestion
        LocalDate endDate = LocalDate.parse(dateTo, DATA_FORMATTER);
```
make sure to format your code before submitting using IDE's autoformatter